src/main/java/seedu/address/model/module/ModuleCode.java

```diff
@@ -128,6 +128,24 @@ public boolean hasTutorialClass(TutorialClass tutorialString) {
         return false;
     }
 
+    /**
+     * List all the tutorial classes under this module.
+     *
+     * @return String of tutorial classes under this module.
+     */
+    public String listTutorialClasses() {
+        if (tutorialClasses.size() == 0) {
+            return String.format("Tutorials in %s: None!", value);
```

Comment:
Has this been tested on the application itself? Just wanna check that value here will not be null, or an empty string.

Reply:
The value is the module code name itself, not related to the tutorial classes.